recipes/fwdpy11/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+
+export CFLAGS="-I${PREFIX}/include"
+export CXXFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+#We must force GCC on OS X
+export CC=gcc
```

Comment:
Let me mess around for another day.  I'm able to build on my OS X machine but not on any of the TravisCI machines with clang.  

Back to it...